aws-logging.yaml

```diff
@@ -0,0 +1,44 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+data:
+  # NEVER set this to true, or set only temporary
+  # it enables internal fluentbit logs that flood Cloudwatch
+  # and cause huge AWS bills 
+  # (see slack post https://this-is-biomage.slack.com/archives/C014YMUT6GN/p1646915799716329)
+  flb_log_cw: "false"
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch
+        Match *
+        region eu-west-1
+        log_group_name fluent-bit-cloudwatch
+        log_stream_name $(kubernetes['pod_name'])
+        auto_create_group true
```

Comment:
Here I use the non-official Fluent Bit output plugin: https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit. The official plugin is called "cloudwatch-logs" and it doesn't work for us, because it doesn't support templating variables (what we do to get the log_stream_name equal to the pod name). 
Link to very useful github issue: https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/issues/16
Link to "cloudwatch_logs" that we didn't use in the end: https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch

This config used to work, but it needs to be double-checked because it started failing at some point. There is a chance that it's not working